aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/receiver/NetworkInfoReceiver.java

```diff
@@ -0,0 +1,126 @@
+package com.amazonaws.mobileconnectors.s3.receiver;
```

Comment:
Great effort!! Just a minor suggestion. Can you move this class to the s3/transferutility package instead of creating a new package?